acbs/utils.py

```diff
@@ -326,3 +326,12 @@ def format(self, record):
                               logging.INFO, logging.DEBUG):
             record.msg = f'[{lvl_map[record.levelname]}]: \033[1m{record.msg}\033[0m'
         return super(ACBSLogFormatter, self).format(record)
+
+def get_defines_file_path(location, stage2=False) -> str:
```

Comment:
I think you should just move this function to `parser.py` since this function is only used there.

And also, add type annotation to this function:

```suggestion
def get_defines_file_path(location: str, stage2=False) -> str:
```